### PR TITLE
Look at new features

### PR DESCRIPTION
**Pull request by SenkaMatic**
I added some new files

### CODE REVIEW

**Comment by DmitriyPermyakov** [comment]
You should add nav section in index.html

File: index.html [hunk 1]
```diff
@@ -1 +1,4 @@
-<body></body>
+<html>
```
Comment (DmitriyPermyakov) [inline index.html:1]: You should add some CSS classes

Comment (SenkaMatic) [reply index.html:1]: Ok, it's will be done.

File: index.html [hunk 2]
```diff
@@ -1 +1,4 @@
...
+<body>
+</body>
+</html>
```
Comment (DmitriyPermyakov) [inline index.html:4]: Add footer

Comment (DmitriyPermyakov) [inline index.html:4]: Add CSS classes to body

File: index.html [hunk 3]
```diff
@@ -1 +1,4 @@
-<body></body>
+<html>
+<body>
+</body>
```
Comment (DmitriyPermyakov) [inline index.html:3]: Add something

**Pull request merged**